client/src/modules/lead/EditLead/EditLeadSidebar/CustomFieldsUtils.ts

```diff
@@ -0,0 +1,41 @@
+import Contact from '../../../../models/Contact';
```

Comment:
Maybe better name is `CustomFieldsService.ts`? Looks like it contains some business logic